packages/mermaid/src/diagrams/info/infoTypes.ts

```diff
@@ -0,0 +1,15 @@
+import type { DiagramDB } from '../../diagram-api/types.js';
+
+export interface InfoFields {
+  info: boolean;
+}
+
+export const DEFAULT_INFO_DB: InfoFields = {
+  info: false,
+} as const;
```

Comment:
This is a value, it shouldn't be in types. 
Let's move it to infoDb.ts

Reply:
I thought the same think, but I said it would be cleaner since it not modifiable constant. I'll move it now.

Reply:
fixed.